index.bs

```diff
@@ -398,6 +401,8 @@ Otherwise, do nothing.
 <emu-alg>
   1. Let _called_ be the number of times _count_ has been invoked (including this invocation) with
   the provided _label_.
+    1. If the associated <a>count map</a> contains an entry with key _label_, <a for="map">set</a> the _value_ of map[_label_] to be map[_label_] + 1.
```

Comment:
This doesn't quite work as "map" is an undefined variable. I think this would work better if you had step 1 be "Let _map_ be the associated count map".

You can then also do `_map_[_label_] <a for="map">exists</a>`